src/waveform/waveformmarklabel.cpp

```diff
@@ -28,7 +28,11 @@ void WaveformMarkLabel::prerender(QPointF bottomLeft, QPixmap icon, QString text
         text = fontMetrics.elidedText(text, Qt::ElideRight, availableWidthForText);
         pixmapRect.setWidth(widgetWidth);
     }
-    pixmapRect.setHeight(math_max(fontMetrics.height(), icon.height()));
+    // We take the pixelSize for background height, to not consider exotic
+    // high characters.
+    pixmapRect.setHeight(
+            math_max(font.pixelSize() * 1.2,
```

Comment:
As far as I can tell this makes no difference. The label is still hidden in Shade when I hover a hotcue.